ci-operator/step-registry/cucushift/aws/upi/OWNERS

```diff
@@ -0,0 +1,3 @@
+approvers:
+- jhou1
+- liangxia
```

Comment:
Please don't forget to add yourself as approver ;)

Reply:
:) Added in latest commit